lib/discourse_activity_pub/content_parser.rb

```diff
@@ -120,7 +120,11 @@ def self.cook(text, opts = {})
 
   def self.scrubbed_html(html)
     doc = Nokogiri::HTML5.fragment(html)
-    scrubber = Loofah::Scrubber.new { |node| node.remove if node.name == "script" }
+    scrubber =
+      Loofah::Scrubber.new do |node|
+        node.remove if node.name == "script"
```

Comment:
Very much unrelated to this PR, but are there other tags we should scrub here? The `style` tag comes to mind.

Reply:
Yes, we probably should. I'll have to think that one through a bit, separately from this PR, as you say. Just that there are a few points at which processing might occur on the `content` (including in `discourse/discourse`).